mne/viz/ica.py

```diff
@@ -160,6 +161,12 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
     hist_ax = var_ax_divider.append_axes("right", size="33%", pad="2.5%")
     var_ax.scatter(range(len(epoch_var)), epoch_var, alpha=0.5,
                    facecolor=[0, 0, 0], lw=0)
+    # rejected epochs in red
+    var_ax.scatter(dropped_indexes, epoch_var[dropped_indexes],
+                   alpha=1., facecolor=[1, 0, 0], lw=0)
+    # compute percentage of dropped epochs
+    var_percent = float(len(dropped_indexes)) / float(len(epoch_var)) * 100.
```

Comment:
if `epoch_var` contains only the retained epochs the the calculation above is incorrect (5 removed / 5 retained would give 100% removed, while it should be 50%)

Reply:
At some point, epoch_var contained all the data so the error comes from here :sweat_smile: I'm changing it !

Reply:
ok, `epoch_var` seems to have all points, so ignore the comment above.